rnote-ui/src/appwindow/appwindowactions.rs

```diff
@@ -537,6 +540,27 @@ impl RnAppWindow {
             }),
         );
 
+        // Remove page from doc in fixed size mode
+        action_remove_page_from_doc.connect_activate(
+            clone!(@weak self as appwindow => move |_action_remove_page_from_doc, _target| {
+                let canvas = appwindow.active_tab().canvas();
+
+                let format_height = canvas.engine().borrow().document.format.height;
+                let format_width = canvas.engine().borrow().document.format.width;
+                let doc_height = canvas.engine().borrow().document.height;
+                let new_doc_height = doc_height - format_height;
+                if doc_height > format_height {
+                    let aabb = Aabb::new(na::point![0.0, new_doc_height], na::point![0.0 + format_width, doc_height]);
+                    canvas.engine().borrow_mut().store.split_colliding_strokes(aabb, aabb);
```

Comment:
this returns WidgetFlags which should be handled. With the suggested changes from the above comment, it could be something like:

```rust
let (_, wf) = canvas.engine().borrow_mut().store.split_colliding_strokes(aabb, aabb);
widget_flags.merge(wf);
```

Reply:
I tried the following:
```
                let mut widget_flags = canvas.engine().borrow_mut().record(Instant::now());  

                let format_height = canvas.engine().borrow().document.format.height;
                let format_width = canvas.engine().borrow().document.format.width;
                let doc_height = canvas.engine().borrow().document.height;
                let new_doc_height = doc_height - format_height;
                if doc_height > format_height {
                    let aabb = Aabb::new(na::point![0.0, new_doc_height], na::point![0.0 + format_width, doc_height]);
                    let (_, wf) = canvas.engine().borrow_mut().store.split_colliding_strokes(aabb, aabb);
                    widget_flags.merge(wf);
                    appwindow.handle_widget_flags(widget_flags, &canvas);
```

but splitting doesn't work anymore...